Automattic/jetpack
# Remove the 'Check for Spam' button from the feedback page

Fixes #27444

#### Changes proposed in this Pull Request:

This PR removes the 'Check for Spam' button from the top of the table on the feedback page when Akismet is enabled.

#### Other information:


#### Jetpack product discussion

pcsBup-oK-p2

#### Does this pull request change what data or activity we track or use?



No.

#### Testing instructions:






- Make sure you have Akismet is installed and active on your test site.
- Go to WP Admin -> Feedback.
- There should no longer be a 'Check for Spam' button at the top of the feedback responses table.
